src/main.cpp

```diff
@@ -117,6 +117,10 @@ static SamApp::ver releases[] = {
 //intermediate version numbers are required in this list in order for the version upgrade script (versions.lk) to work correctly
 //please clarify the reason for the new version in a comment. Examples: public release, variable changes, internal release, public beta release, etc.
 //the top version should always be the current working version
+		{ 2021, 4, 19 }, // 2021.4.19 ssc 255 EPRI Beta expires 4/19/2022
```

Comment:
Most recent version should be for the patch, correct? It looks like develop needs to be merged into this branch. I wonder if a command-line merge would solve the Travis issues as well.

Reply:
This branch was made off of develop. And is kept up to date with develop. This includes the latest beta builds. Patch does not include them. Darice usually merges patch into develop after a PySAM update.